vpr/src/route/route_net.tpp

```diff
@@ -0,0 +1,520 @@
+#pragma once
+
+/** @file Header implementations for templated net routing fns. */
+
+#include <tuple>
+#include "route_net.h"
+
+#include "connection_router_interface.h"
+#include "describe_rr_node.h"
+#include "draw.h"
+#include "route_common.h"
+#include "route_debug.h"
+#include "route_profiling.h"
+#include "rr_graph_fwd.h"
+
+template<typename ConnectionRouter>
+inline NetResultFlags try_timing_driven_route_net(ConnectionRouter& router,
```

Comment:
Comment what this routine does (and how is it different than timing_driven_route_net).  Or just merge them (you say merging is your plan).